ChromecastSampleApp/app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,6 @@
   <string name="hello">Hello World, ChromecastSampleAppActivity!</string>
   <string name="app_name">Ooyala Chromecast Sample App</string>
   <string name="media_route_menu_title">Play on..</string>
+  <string name="loading">Loading</string>
+  <string name="castingTo">Casting to   </string>
```

Comment:
I felt the device name is too close to "casting to " part. Maybe you are right, I'll try to use a format string instead.